Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -95,14 +92,6 @@ struct PreplannedListItemView: View {
                 self.downloadState = downloadState
             }
         }
-        .task(id: downloadState) {
-            if previousDownloadState == .notDownloaded && downloadState == .downloading {
-                onDownloadAction?()
-            }
-            if previousDownloadState == .downloaded && downloadState == .notDownloaded {
```

Comment:
it's not great to check the state for determining if it was removed. The reason being is that we have a function to remove the download so we can know for certain when it happens.